### DISCUSSION TITLE
Welcome, Trevor Wang!

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#329] [Subscribe]
Ahoy @trevorwang, welcome aboard!

Thank you for joining us! Every day there are new quiz questions posted in [Discussions](https://github.com/ParanoidUser/yolo/discussions/categories/galaxy-trivia), you can try to answer them to earn the Galaxy Brain badge. Spread the word, collaborate and have fun, have fun, have fun! 😊

Let me know if you have any questions.

**Comment by trevorwang** [comment]
@ParanoidUser  Thanks for the invitation.

Lets do it.

**Comment by ParanoidUser** [comment]
Regarding the **YOLO** (https://github.com/ParanoidUser/yolo/pull/330), don't forget to set a reviewer (anyone) before merging. 😉

**Comment by ParanoidUser** [comment]
Hey @trevorwang! If you want to try again, here is the step-by-step guide for YOLO https://github.com/ParanoidUser/yolo/discussions/18, important **step #4** to assign any reviewer before mering.
